src/lib/gw2/itemStats.spec.js

```diff
@@ -0,0 +1,199 @@
+import applyAttributes from './itemStats';
+
+// New Stat (Quad Attrib)
+const STAT_MINSTREL = {
+  id: 1134,
+  name: 'Minstrel\'s',
+  attributes: {
+    Toughness: 0.3,
+    Vitality: 0.165,
+    Healing: 0.3,
+    BoonDuration: 0.165,
+  },
+};
+
+// Celectiel stat (7 Attrib)
+const STAT_CELECTIEL = {
+  id: 588,
+  name: 'Celestial',
+  attributes: {
+    Power: 0.165,
+    Precision: 0.165,
+    Toughness: 0.165,
+    Vitality: 0.165,
+    CritDamage: 0.165,
+    Healing: 0.165,
+    ConditionDamage: 0.165,
+  },
+};
+
+// Berseker (3 Attrib - Legacy)
+const STAT_BERSEKER = {
+  id: 161,
+  name: 'Berserker\'s',
+  attributes: {
+    Power: 0.35,
+    Precision: 0.25,
+    CritDamage: 0.25,
+  },
+};
+
+const STAT_BERSEKER_VALKYRE = {
+  id: 591,
+  name: 'Berserker\'s and Valkyrie',
+  attributes: {
+    Power: 0.35,
+    Precision: 0.25,
+    Vitality: 0,
+    CritDamage: 0.25,
+  },
+};
+
+const STAT_WIRED = {
```

Comment:
yeah figured as much, all good!

merged and deployed to prod

fyi i brought in deploy to prod only for tagged commits, will make things a little easier from now on :)